lessons/PowerShell/managed-identity-powershell.md

```diff
@@ -0,0 +1,240 @@
+# Managed Identity (PowerShell)
+
+Watch the recording of this lesson [on YouTube]().
+
+## Goal 🎯
+
+The goal of this lesson is to understand how you can create and use a system assigned managed to call an Azure function in order to obtain a Microsoft Graph access token with the right permission scope. Microsoft Graph is THE API for all things Microsoft 365. 
```

Comment:
```suggestion
The goal of this lesson is to understand how you can create and use a system assigned managed identity to call an Azure Function in order to obtain a Microsoft Graph access token with the right permission scope. Microsoft Graph is THE API for all things Microsoft 365. 
```

Reply:
I suggest you add a sentence why managed identities are preferred over secrets. And a link to https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview would be welcome as well. The link can also be added to the _More Info_ section.

Reply:
✅